src/modules/interfaceAPI.js

```diff
@@ -0,0 +1,25 @@
+import { readFromForm, populateScoresTable, showMessage } from './appCRUD.js';
+
+const gameId = '/xAnLxnpCzCYPPY9fdvNC';
+const baseUrl = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/';
+const scoreUrl = `${baseUrl}games${gameId}/scores/`;
+
+export const fetchFromAPI = async () => {
+  const scores = await fetch(scoreUrl).then((response) => response.json()).then((json) => json);
+  populateScoresTable(scores.result);
+};
+
+export const writeToAPI = async () => {
+  const userInput = readFromForm();
+  if (userInput !== null) {
+    fetch(scoreUrl, {
+      method: 'POST',
+      body: JSON.stringify(userInput),
+      headers: {
+        'Content-type': 'application/json; charset=UTF-8',
+      },
+    })
+      .then((response) => response.json());
+    showMessage('score added successfully', 'green');
+  }
```

Comment:
[REQUIRED] It is a requirement to use async and await. Kindly fix this.

Reply:
resolved! ✔️

Reply:
Hi @fobadara I have implemented the changes. Thank you for the feedback